package.json

```diff
@@ -4,22 +4,25 @@
   "private": true,
   "dependencies": {
     "@capacitor/core": "1.3.0",
-    "@ionic/react": "^5.3.1",
-    "@ionic/react-router": "^5.3.1",
+    "@ionic/react": "^6.0.0-rc.2",
+    "@ionic/react-router": "^6.0.0-rc.2",
     "@types/jest": "24.0.18",
     "@types/node": "12.7.5",
     "@types/react": "^16.9.43",
     "@types/react-dom": "^16.9.8",
     "@types/react-router": "^5.1.8",
     "@types/react-router-dom": "^5.1.5",
+    "date-fns": "^2.25.0",
     "ionicons": "^5.0.1",
+    "mini-create-react-context": "^0.4.1",
```

Comment:
What is this dependency for?

Reply:
It came up as a missing peer dep while I was upgrading React, although I just tried removing it and things seem okay 🤷‍♀️ Went ahead and ditched it.